notebooks/Notebook3.md

```diff
@@ -9,6 +9,14 @@ Did it help you make a design decision? Was it helpful in addressing the most pr
 issues in your project? How will you incorporate the feedback into your work? Will you
 change something about the design, implementation, or evaluation as a result?
 
+The critiques I received last week set me on the right track with some of the design issues I was having,
+especially my stylistic issues. Adam has been keeping me on track with not getting bogged down in the style of 
+my language and focusing on functionality over form for the time being. Checking out the rules of Magic: The 
+Gathering has also been good, as it's been making me think about how game designers think about timing windows 
+during games, and how blocks of code might be reused, e.g. since MTG "Main Phases" happen twice per turn. Our 
+discussion of parentheses-less and infix functions in class was also super helpful, but I anticipate I'll end 
+up thinking about that more when I finish the main functionality.
```

Comment:
Im glad my advice helped! I know you were stuck with some errors but keep pushing! I think your almost to something really cool.